.env.template

```diff
@@ -1,9 +0,0 @@
-# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
```

Comment:
Rather than deleting this, I'm a fan of keeping it and maintaining as environment variables change - it's a really convenient way for new devs to know how to be able to quickly set up their environment

Reply:
agree. I think I did `mv ...` rather than `cp ...` and then rushed on without thinking